### PR TITLE
[DPN-24/Localstorage] add icon for export data to json/copy current l…

### PR DESCRIPTION
**Pull request by Yifan-Zhang618**
…ink and an input file area for uploading .json file

add 2 icons and 1 file input section
icon#1: click for exporting data as a JSON file, on hover for its description
icon#2: click for copying the current link, on hover for its description
file input section: upload JSON file as input data

### CODE REVIEW

File: static/js/localStorage.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        if(element.value === '' ) {
+            input.value = 0;
+        }
+        else input.value = element.value;
+
+    });
+
+    let myJSON = JSON.stringify(inputs);
+    text = myJSON;
+    let filename = "input.json";
```
Comment (trevorliberty) [inline static/js/localStorage.js:33]: Let's rename this to "parameters.json"

**Review by trevorliberty — CHANGES_REQUESTED** [review]
Only minor changes needed.
Make sure you write unit tests for these changes as well

**Comment by Yifan-Zhang618** [comment]
I Will add a unit test  later when I figure it out

**Comment by jjesse55** [comment]
Make sure to pull in the latest changes from master as their was a PR that got merged last night

**Review by trevorliberty — COMMENTED** [review]
Let's add a test for downloading and copying the link successfully as well

**Pull request closed without merging**